pkgs/applications/kde/spectacle.nix

```diff
@@ -14,7 +14,7 @@ mkDerivation {
   buildInputs = [
     kconfig kcoreaddons kdbusaddons kdeclarative ki18n kio knotifications
     kscreen kwidgetsaddons kwindowsystem kxmlgui libkipi qtx11extras xcb-util-cursor
-    knewstuff kwayland
+    knewstuff kwayland kcolorpicker kimageannotator
   ];
   postPatch = ''
     substituteInPlace desktop/org.kde.spectacle.desktop.cmake \
```

Comment:
Please set meta

```nix
  meta = with lib; {
    broken = versionOlder qtbase.version "5.15";
  };
```

Reply:
Thanks for the suggestion. I'm not sure why @r-rmcgibbo tried to build `libsForQt514.spectacle`, it was already marked as broken *without* this meta setting.